backend/src/main/java/com/google/sps/utils/UserCollector.java

```diff
@@ -56,4 +56,10 @@ public static void addRecipeToUserRecipeList(
       datastore.put(userEntity);
     }
   }
+
+  public static void addInventoryToUser(Entity userEntity, List<String> inventory) {
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+    userEntity.setProperty(UserConstants.PROPERTY_INVENTORY, inventory);
+    datastore.put(userEntity);
```

Comment:
I don't see the use of this method anywhere else but InventoryServlet. Since it's just 3 lines, maybe add them directly to the post request?